### ISSUE TITLE
Update `remote` documentation?

### ISSUE THREAD

**Issue by jprichardson** [atom/electron#1748]
In the `remote` documentation:

> In Electron, only GUI-related modules are available in the renderer process.

Is this still true? If not, when did this change? In a `renderer` process, I'm able to `require` any Node.js module, 3rd party module, and my own code - directly from the dev tools pane. Am I doing something wrong?

From this example on the `remote` documentation page:

```js
var remote = require('remote');
remote.getCurrentWindow().capturePage(function(buf) {
  remote.require('fs').writeFile('/tmp/screenshot.png', buf, function(err) {
    console.log(err);
  });
});
```

Why even `remote.require('fs')` when you could just do `require('fs')`?

**Comment by jprichardson** [comment]
Oops, forget my question about the example, I missed the context of `Buffer` in the discussion. But original question still stands about the statement:

> In Electron, only GUI-related modules are available in the renderer process.

**Comment by zcbenz** [comment]
It is a typo, it should be "GUI-unrelated".

**Comment by zcbenz** [comment]
Fixed by 33a926c.
